pkg/secrethub/main_test.go

```diff
@@ -40,19 +42,20 @@ func init() {
 // setup starts a test server and returns a router on which tests can register handlers.
 // Tests should use the returned client options to create new Clients and should call the
 // cleanup func() when done.
-func setup() (chi.Router, *ClientOptions, func()) {
+func setup() (chi.Router, []ClientOption, func()) {
 	// router is the HTTP router used with the test server.
 	router := chi.NewRouter()
 
 	// Strip prefixes so tests can register routes on e.g. /users instead of /v1/users.
 	handler := http.NewServeMux()
-	handler.Handle(baseURLPath+"/", http.StripPrefix(baseURLPath, router))
+	handler.Handle("/v1/", http.StripPrefix("/v1", router))
 
 	// server is a test HTTP server used to provide mock API responses.
 	server := httptest.NewServer(handler)
 
-	opts := &ClientOptions{
-		ServerURL: server.URL,
+	opts := []ClientOption{
+		WithServerURL(server.URL),
+		WithCredentials(cred1),
```

Comment:
Do we want to use the cred1 by default and overload it when we need a client for another account or do we want to be explicit about it in every test?

Reply:
Makes sense to use `cred1` by default, we use it everywhere :astonished: